package.json

```diff
@@ -25,10 +25,10 @@
     "png",
     "svg"
   ],
-  "author": "Nico Rehwaldt <https://github.com/nikku>",
+  "author": "Nico Rehwaldt <https://github.com/nikku>, Ruslan Gainutdinov <https://github.com/huksley>",
```

Comment:
Please remove yourself from the authors. You may add yourself to the list of contributors, cf. [npm documentation](https://docs.npmjs.com/files/package.json#people-fields-author-contributors) if you'd like to.

Reply:
Removed